core/src/main/java/hudson/model/Job.java

```diff
@@ -1042,7 +1047,82 @@ public PermalinkList getPermalinks() {
         }
         return permalinks;
     }
-    
+
+    /**
+     * RSS feed for changes in this project.
+     */
+    public void doRssChangelog(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException {
+        class FeedItem {
+            ChangeLogSet.Entry e;
+            int idx;
+
+            public FeedItem(ChangeLogSet.Entry e, int idx) {
+                this.e = e;
+                this.idx = idx;
+            }
+
+            Run<?, ?> getBuild() {
```

Comment:
Plus it'd theoretically be changing an API, since this got moved over from `AbstractBuild`.